edb/common/debug.py

```diff
@@ -75,6 +75,9 @@ class flags(metaclass=FlagsMeta):
     bootstrap = Flag(
         doc="Debug server catalog bootstrap.")
 
+    cache_yolo = Flag(
```

Comment:
perhaps `bootstrap_cache_yolo` or do you mean for this to apply to all cache everywhere?

Reply:
Good question. Is `edb.server.buildmeta.read_data_cache` used for other caches? (I don't really know what all caches we have.)

Reply:
`read_data_cache` is only used for stdlib cache in devmode (server and also serverless schema tests). I think it's fine to name it `bootstrap_cache_yolo`